bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -265,25 +285,73 @@ message SwapExactInResponse {
 message WithdrawQuotesRequest {
   // base_asset: The base asset against which the accumulated quotes that the user wants to withdraw. e.g. USDT pool
   string base_asset = 1;
-}
-
-// WithdrawQuotesResponse: Response to the WithdrawQuotesRequest. Contains information about the withdrawn quotes.
+  // deposit_address: The address where withdrawn quotes should be deposit. If not provided, the default sender address
+  // would be used. Note it can be overwritten for specific quotes.
+  optional string deposit_address = 2;
+  // quotes: Details of how specific quotes should be withdrawn. Note that quotes that are not valid for this base, or
+  // were not collected would not cause an error - they will be simply ignored.
+  repeated QuoteWithdrawal quotes = 3;
+  // include_all_quotes: Determines if the quotes that are not included in `quotes` but were collected for this base
+  // should be also withdrawn using default withdrawal settings. It might be usefull to withdraw all the collected quotes,
+  // but specifying some details like deposit address only to limited subset of quotes. Setting this field with no `quotes`
+  // specified would result in error - when `quotes` are skipped, all the quotes are returned, and we need to make sure this
+  // field is not set to `false` accidently in which case the message would have no sense.
+  bool include_all_quotes = 4;
```

Comment:
we could infer this if the above field is empty.

Reply:
The idea of that is to allow withdrawing all the quotes, but specifying the custom receiver for only some of them. But in context of your prev comment that is not needed anyway.